lib/plugins/authad/action.php

```diff
@@ -73,17 +73,33 @@ public function handleHtmlLoginformOutput(Doku_Event $event, $param)
 
             // update user field value
             if ($dom) {
-                $usr          = $auth->getUserName($usr);
-                $pos          = $form->findElementByAttribute('name', 'u');
-                $ele          =& $form->getElementAt($pos);
-                $ele['value'] = $usr;
+                $usr = $auth->getUserName($usr);
+                if (is_a($form, 'dokuwiki\Form\Form')) {
```

Comment:
There is no need to implement fallbacks for bundled plugins - they are in sync with the core code so they will always get the new form

Reply:
fixed in 903616e